pkg/tlshelpers/tlshelpers.go

```diff
@@ -135,3 +136,14 @@ func getCommonNameFromOrg(org string) (string, error) {
 	}
 	return "", errors.New("unable to recognise secret name")
 }
+
+func GetDefaultEtcdCipherSuites() string {
```

Comment:
I think you're looking for a configobserver: https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/configobservation/configobservercontroller/observe_config_controller.go#L122 .

Using that and writing to your observedConfig location will allow you to rollout the change with an assigned generation and react to changes as the platform changes security posture.  And it leaves the cluster-admin with a single spot to configure all tls